.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     working_directory: ~/TensorComprehensions
     resource_class: xlarge
     docker:
-      - image: tensorcomprehensions/linux-trusty-gcc4.8-cuda8-cudnn6-py3-conda:x86
+      - image: tensorcomprehensions/linux-trusty-gcc4.9-cuda8-cudnn7-py3-conda:1
```

Comment:
Does this bump the required gcc from 4.8.5 to 4.9 ?
If so, please document and explain why.

Reply:
reason is in the commit message. Where else do you recommend to document this?